TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m

```diff
@@ -0,0 +1,23 @@
+//
+//  TSDKSmsGateway.m
+//  TeamSnapSDK
+//
+//  Created by Skyler Seamans on 12/5/16.
+//  Copyright © 2016 teamsnap. All rights reserved.
+//
+
+#import "TSDKSmsGateway.h"
+
+@implementation TSDKSmsGateway
+
+@dynamic domain, name;
+
++ (NSString *)SDKType {
+    return @"SmsGateway";
+}
+
++ (NSString *)SDKREL {
+    return @"sms_gateways";
```

Comment:
one SDKType is sms_gateway SDKRel isn't needed. it's only needed when the plural isn't formed by adding an S